src/components/Card.jsx

```diff
@@ -0,0 +1,136 @@
+import Arrow from "../assets/icon-arrow.svg";
+import { useEffect, useState } from "react";
+
+import FormInput from "./FormInput";
+import AgeResults from "./AgeResults";
+const monthsUtil = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
+const Card = () => {
+  const [day, setDay] = useState("");
+  const [month, setMonth] = useState("");
+  const [year, setYear] = useState("");
+  const [result, setResult] = useState({
+    year: "-",
+    month: "-",
+    days: "-",
+  });
+  const [error, setError] = useState({
+    error: false,
+    msj: "",
+  });
+
+  useEffect(() => {
+    if (day.length == 0 || month.length == 0 || year.length == 0) {
```

Comment:
Not sure about this. If error [is derived from a state it probably doesn't need to *be* a state](https://legacy.reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html). Same thing for the `useEffect`, not sure that you have to trigger it since the trigger is an actual state.